### PR TITLE
QOL improvements

### PR DESCRIPTION
**Pull request by tbodt**
- have the right click command default to quick edit instead of having to click a separate button (quality of life)
- include thread_id url parameter on webhook when restoring a message from a thread (bugfix)

### CODE REVIEW

File: src/lib/messages/fetchAndRestoreMessage.ts [hunk 1]
```diff
@@ -84,25 +87,10 @@ export const fetchAndRestoreMessage = async (
...
-            `will expire ${time(new Date(response.expires), "R")}.`,
-        },
-      ],
-      components,
-    })
-    return
-  }
-
-  if (!webhook) {
+    message = message
```
Comment (ghost) [inline src/lib/messages/fetchAndRestoreMessage.ts:90]: lol

**Review by ghost — COMMENTED** [review]
this code base is dead and on life support right now but I'll merge this anyway

File: src/lib/messages/fetchAndRestoreMessage.ts (hunk 1 above)
Comment (tbodt) [reply src/lib/messages/fetchAndRestoreMessage.ts:90]: hah i missed this .

**Pull request merged**